sled-agent/src/params.rs

```diff
@@ -485,6 +485,20 @@ impl From<ServiceZoneRequest> for sled_agent_client::types::ServiceZoneRequest {
     }
 }
 
+#[derive(
+    Clone, Debug, Deserialize, Serialize, JsonSchema, PartialEq, Eq, Hash,
+)]
+pub struct ServiceZoneService {
```

Comment:
I can probably do another pass on the naming here to collapse "Service" out of this in a separate PR, but the addition of this structure makes sense to me!

Reply:
Thanks for taking a look!